### PR TITLE
Fix dependency packages

### PR DESCRIPTION
**Pull request by weshaggard**
cc @mmitche @ericstj

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -4,22 +4,13 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
   </PropertyGroup>
-  <!-- Package names -->
-  <PropertyGroup>
```
Comment (mmitche) [inline eng/Versions.props]: You can leave this if you want.  I added support for creating the property group in my latest arcade PR.

**Comment by mmitche** [comment]
@weshaggard Since a bunch of other repos use PackageVersion by default, and I'd like this to be consistent across repos, it's probably better to put in a temporary set of Version properties, and set the PackageVersion off of those.  When we rollout and make this configurable, then I'll just remove those

**Comment by mmitche** [comment]
@weshaggard core-sdk, coreclr, core-setup, aspnet, etc. all use 'PackageVersion' today.

File: eng/Versions.props (hunk 1 above)
Comment (mmitche) [reply eng/Versions.props]: Currently the default xmlns https://github.com/dotnet/corefx/pull/33014/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR2 will cause the updates not to happen here.  Fixed in https://github.com/dotnet/arcade/pull/1140

**Comment by weshaggard** [comment]
test Linux arm64 Release Build

**Pull request merged**